pkg/recorder/recorder.go

```diff
@@ -21,11 +21,15 @@ limitations under the License.
 package recorder
 
 import (
+	"k8s.io/client-go/tools/events"
 	"k8s.io/client-go/tools/record"
 )
 
 // Provider knows how to generate new event recorders with given name.
 type Provider interface {
 	// NewRecorder returns an EventRecorder with given name.
-	GetEventRecorderFor(name string) record.EventRecorder
+	GetEventRecorder(name string) events.EventRecorder
```

Comment:
Why rename this?

Reply:
I think that suggestion was to aid in making this a non-breaking change

Reply:
I interpreted it as a general suggestion, my bad. I will revert that change.

Reply:
Ah yeah. I only meant if we introduce another func, might as well improve the name